src/api/fragment.js

```diff
@@ -29,10 +29,15 @@ function decorateFragmentMethods (frag) {
 }
 
 export default function (html) {
-  var frag = document.createDocumentFragment();
+  const frag = document.createDocumentFragment();
   decorateFragmentMethods(frag);
   if (typeof html === 'string') {
-    var par = createFromHtml(html);
+    let par = createFromHtml(html);
+
+    if (par.nodeType !== 1) {
```

Comment:
@nicolasv what do you think about this? Before, we couldn't use `skate.fragment()` for templating unless it had a wrapper element. I'm not sure if we should do it this way or use `par.firstChild` instead of `par.firstElementChild` so that all node types are captured.